files/generate-CA-and-clientcerts.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
something in this script could be busted, causing the SSL error messages later.  I could use another pair of eyeballs on this by someone more familiar with SSL than me.

this will generate a series of certs/keys which will be used in a later step by nginx and the clients